plenty_api/api.py

```diff
@@ -824,7 +825,69 @@ def plenty_api_set_image_availability(self,
                                              path=path,
                                              data=data)
 
-        if not response:
-            return False
+        return response
 
-        return True
+# PUT REQUESTS
+
+    def book_incoming_items(self,
```

Comment:
The naming convention of the function can be open for discussion but for now, I would fix those names during the merge in order to align with the rest of the API interface. 
Meaning:
s/book_incoming_items/plenty_api_book_incoming_items/
s/book_outgoing_items/plenty_api_book_outgoing_items/

My reason for this convention was for the methods to be as descriptive as possible while being used within a user application. (Especially making clear when the plenty API is touched)
I would rather say, if we agree on a new naming convention for the methods then we do it for all the public methods for v.0.2.8.

Reply:
You are right, we should have named them according to the convention. Would you prefer me to fix it or to do it yourself?

Reply:
I have renamed them now.